### DISCUSSION TITLE
Use RTL instead of Enzyme to generate Unit Tests

### DISCUSSION THREAD

**Discussion by sameera** [JudiniLabs/code-gpt-docs#68] [Ideas]
The `Unit Test Code GPT` command generates unit tests that use Enzyme which is incompatible with React 18. 
It would be good to default to RTL instead OR better, if it looked at the `package.json` to figure out the testing libraries used and generate the tests accordingly.

**Comment by davila7** [comment]
That is an excellent idea!
I'm going to check if I can get access to package.json and get that information.
